util/maven/generate_poms.py

```diff
@@ -58,6 +58,29 @@ def pom_deps(label):
         'name': 'Dagger',
         'artifact': 'dagger',
     },
+    '//java/dagger:gwt': {
+        'name': 'Dagger GWT',
+        'artifact': 'dagger-gwt',
+        'manual_dependencies': '''
+            <dependency>
+              <groupId>%s</groupId>
```

Comment:
Do we need javax.inject non-sources deps too?

Either way, the regular deps can be just on the `deps` attributes for the `:gwt` target, right?